.github/workflows/ant.yml

```diff
@@ -0,0 +1,22 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
```

Comment:
Q: 
- If we want to run test against other Java versions, is this where we bump the number in the future?
- Is Java 11 minimum now?

Reply:
You can bump this version yes, or add copies of this file to test other versions as well (If I understand github actions correctly). For convenience I would say Java 11 is a minimum yes. I don't expect we have any 'legacy' apps for ipfs (We are already up to Java 14, so 11 itself is starting to age).